bucket/zim.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version":  "0.75.1",
+    "license":  "GPL-2.0-only",
+    "description":"Zim is a graphical text editor used to maintain a collection of wiki pages. Each page can contain links to other pages, simple formatting and images. Pages are stored in a folder structure, like in an outliner, and can have attachments. Creating a new page is as easy as linking to a nonexistent page. All data is stored in plain text files with wiki formatting. Various plugins provide additional functionality, like a task list manager, an equation editor, a tray icon, and support for version control.",

```

Comment:
This description seems longer than usual. Our [docs](https://github.com/ScoopInstaller/Scoop/wiki/App-Manifests#required-properties) say it should be "one line." I would interpret this to mean one sentence, or two small sentences, at most. Would you mind shortening this? Perhaps:
```json
    "description":"A graphical text editor to maintain wiki pages.",
    "##": "Each page can contain links to other pages, simple formatting and images. Pages are stored in a folder structure, like in an outliner, and can have attachments. Creating a new page is as easy as linking to a nonexistent page. All data is stored in plain text files with wiki formatting. Various plugins provide additional functionality, like a task list manager, an equation editor, a tray icon, and support for version control.",
```

Reply:
You are right. I commited the shorter description. Thanks.